app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -60,9 +60,9 @@
   display: inline-block;
   margin-top: .1em;
   text-decoration: underline;
-  margin-left: $gutter;
+  margin-left: govuk-spacing(5);
```

Comment:
I thought `$gutter` was govuk-spacing 6, not 5?

Reply:
See comment below. We switched from desktop first to mobile first that's why the values are the other way around.